Documentation/devicetree/bindings/reserved-memory/linaro,secure-heap.yaml

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: (GPL-2.0 OR BSD-2-Clause)
+%YAML 1.2
+---
+$id: http://devicetree.org/schemas/reserved-memory/linaro,secure-heap.yaml#
+$schema: http://devicetree.org/meta-schemas/core.yaml#
+
+title: Linaro Secure DMABUF Heap
+
+maintainers:
+  - Olivier Masse <olivier.masse@nxp.com>
+
+description:
+  Linaro OP-TEE firmware need a reserved memory for the
+  Secure Data Path feature.
```

Comment:
It would be nice to explain a bit more about SDP here (its purpose and how it achieves it in a few words).

Reply:
Please find the description here
https://github.com/linaro-swg/linux/pull/101/commits/d04ae897164612b6b152c8d292844fca6bb066ae